x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_notify_when.tsx

```diff
@@ -118,6 +118,49 @@ export const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<RuleNotifyWhenType>
   },
 ];
 
+const SIEM_NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<RuleNotifyWhenType>> = [
```

Comment:
These should be defined in the `security_solution` plugin code, and passed into the action form as a prop.

Reply:
done